### PR TITLE
Add NUTS fields

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

**Review by costezki — COMMENTED** [review]
/prbot describe

**Comment by costezki** [comment]
copilot:all

File: tests/unit/test_nuts_generation.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from ted_data_eu.services.etl_pipelines.ted_data_etl_pipeline import generate_nuts_level
+
+
+def test_nuts_generation(real_nuts, fake_nuts):
```
Comment (costezki) [inline tests/unit/test_nuts_generation.py:4]: Surely there are more cases of NUTS. 
Can you try Copilot to write the assertions for you if you give it a few examples of NUTS codes?

**Review by costezki — APPROVED** [review]

**Review by CaptainOfHacks — APPROVED** [review]

**Pull request merged**